processing/requirements.txt

```diff
@@ -8,3 +8,5 @@ redis==2.10.6
 requests==2.18.4
 scrapy==1.5.0
 SQLAlchemy==1.2.5
+scrapy-splash==0.7.2
+ipython
```

Comment:
ipython is required for development. I don't think it needs to be included as something needed in production.